### ISSUE TITLE
Instant-cast mounts should take priority over falling protection

### ISSUE THREAD

**Issue by mbattersby** [xod-wow/LiteMount#78]
https://www.curseforge.com/wow/addons/litemount?comment=1563

> Hello, this is probably low priority, but during the event of Hallow's End (Halloween), could the default options set the Magic Broom to be higher priority than Levitate, Zen Flight, etc., in areas that allow for flying? Currently, when I jump off places, Litemount will prioritize using those class abilities over the broom, and it's slightly annoying. Would be a nice minor QOL change.
> Thank you!

**Comment by mbattersby** [comment]
This is my current best guess but need to check it against Flight Form.
```
LeaveVehicle
Dismount [nofalling]
CopyTargetsMount
ApplyRules
Limit [mod:shift,nosubmerged,flyable] RUN,~FLY
Limit [mod:shift,submerged] -SWIM
SmartMount
IF [falling]
  # Slow Fall, Levitate, Zen Flight, Glide, Flap
  Spell 130, 1706, 125883, 131347, 164862
  # Hearty Dragon Plume, Rocfeather Skyhorn Kite
  Use 182729, 131811
  # First dismount is [nofalling]
  Dismount
END
Macro
```

**Comment by mbattersby** [comment]
Fixed in 92e52249fa095d3d6241ff8470508aa2d7b6d815

**Comment by mbattersby** [comment]
Released in 9.1.22, closing.
